node-red-node-wot/README.md

```diff
@@ -11,7 +11,7 @@ The package is built upon the [node-wot](https://github.com/eclipse-thingweb/nod
 
 ## Provided Nodes
 
-After installation, the package adds eight nodes at the Node-RED palette, each scoped under the __Web of Things__ title.
+After installation, the package adds ten nodes at the Node-RED palette, each scoped under the __Web of Things__ title.
```

Comment:
I suggest removing any number


```suggestion
After installation, the package adds some nodes at the Node-RED palette, each scoped under the __Web of Things__ title.
```

OR 


```suggestion
After installation, the package adds nodes at the Node-RED palette, each scoped under the __Web of Things__ title.
```

Reply:
Total rewrite for better readability and clarity:
`After installation, the package adds several nodes to the Node-RED palette, all grouped under the Web of Things title. These nodes are designed to interact with the various interaction affordances of a Thing:`